### PR TITLE
git modules: shallow and fix branch

### PR DESCRIPTION
**Pull request by JekaMas**
# Description

Please provide a detailed description of what was done in this PR

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [x] I have updated the official documentation
- [x] I have added sufficient documentation in code

## Testing

- [x] I have tested this code with the official test suite
- [x] I have tested this code manually

### Manual tests

Cloned from scratch and compiled

### CODE REVIEW

**Comment by JekaMas** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by b00ris — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,9 @@
 [submodule "tests/tests"]
 	path = tests/tests
-	url = https://github.com/ethereum/tests.git 
+	url = https://github.com/ethereum/tests.git
+	shallow = true
 [submodule "core-contracts"]
 	path = core-contracts
 	url = https://github.com/0xPolygon/core-contracts
+	shallow = true
+	branch = dev-ethernal
```
Comment (Stefan-Ethernal) [inline .gitmodules]: This is not the branch we are running for release, but it is `feat-polybft-release` instead (https://github.com/0xPolygon/core-contracts/tree/feat-polybft-release).

**Review by Stefan-Ethernal — APPROVED** [review]
Just fix branch for the core-contracts, otherwise lgtm

File: .gitmodules (hunk 1 above)
Comment (b00ris) [reply .gitmodules]: Just checked and wanted to mention the same)

Comment (JekaMas) [reply .gitmodules]: done

Comment (JekaMas) [reply .gitmodules]: @Stefan-Ethernal @b00ris

**Pull request merged**